README.md

```diff
@@ -174,6 +174,36 @@ If you use a value other than `$(PRODUCT_BUNDLE_IDENTIFIER)` in the `CFBundleURL
 
 > For more info please read the [React Native docs](https://facebook.github.io/react-native/docs/linking.html).
 
+#### Expo
+
+> :warning: This SDK is not compatible with "Expo Go" app because of custom native code. It is compatible with Custom Dev Client and EAS builds
+
+To use the SDK with Expo, we need to configure the app at build time. We need to provide the `domain` and optionally the `customScheme` value through the [Config Plugin](https://docs.expo.dev/guides/config-plugins/) by adding the following snippet to _app.json_ or _app.config.js_
+
+```json
+{
+  "expo": {
+    ...
+    "plugins": [
+      [
+        "react-native-auth0",
+        {
+          "domain": string,
+          "customScheme": string (optional),
```

Comment:
Wonder if it's better to use placeholder values here that we use elsewhere. e.g.:

```suggestion
          "domain": "YOUR_AUTH0_DOMAIN",
          "customScheme": "YOUR_CUSTOM_SCHEME",
```